app/repositories/assessmentcentre/AssessmentEventsRepository.scala

```diff
@@ -28,6 +28,7 @@ import scala.concurrent.Future
 
 trait AssessmentEventsRepository {
   def save(events: List[Event]): Future[Unit]
+  def fetchEvents(eventTypeOpt: Option[String], venueTypeOpt: Option[String]) : Future[List[Event]]
```

Comment:
Yes, please make these enums.  We can also default these values to None so that if we want to query all the events the function sig is a little less verbose.